wfe/wfe.go

```diff
@@ -329,6 +331,54 @@ func (wfe *WebFrontEndImpl) handleKey(
 	}
 }
 
+func (wfe *WebFrontEndImpl) handleCertStatusBySerial(
+	ctx context.Context,
+	response http.ResponseWriter,
+	request *http.Request) {
+
+	serialStr := strings.TrimPrefix(request.URL.Path, certStatusBySerial)
+	serial := big.NewInt(0)
+	if _, ok := serial.SetString(serialStr, 16); !ok {
+		response.WriteHeader(http.StatusNotFound)
```

Comment:
nit: in this case I think it makes more sense to return `http.StatusBadRequest` instead of `http.StatusNotFound` since it indicates the serial wasn't a valid hex encoded bigint.
```suggestion
		response.WriteHeader(http.StatusBadRequest)
```

Reply:
Good point!